WPThemeReview/Tests/CoreFunctionality/PrefixAllGlobalsUnitTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Unit test class for WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Tests\CoreFunctionality;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+/**
+ * Unit test class for the PrefixAllGlobals sniff.
+ *
+ * @package WPTRT\WPThemeReview
+ *
+ * @since 0.2.0
+ */
+class PrefixAllGlobalsUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Error files with the expected nr of errors.
+	 *
+	 * @var array
+	 */
+	private $expected_results = array(
```

Comment:
This array can be dropped.

Instead of testing that we are receiving errors, we are testing that errors will _not_ be thrown.